admin/app/model/AdminLifecycle.scala

```diff
@@ -89,13 +89,15 @@ class AdminLifecycle(appLifecycle: ApplicationLifecycle,
       matchDayRecorder.record()
     }
 
-    if (environment.isProd) {
-      val londonTime = TimeZone.getTimeZone("Europe/London")
-      jobs.scheduleWeekdayJob("ExpiringSwitchesEmailJob", 48, 8, londonTime) {
-        log.info("Starting ExpiringSwitchesEmailJob")
-        ExpiringSwitchesEmailJob(emailService).run()
-      }
+    val londonTime = TimeZone.getTimeZone("Europe/London")
```

Comment:
Not exactly - I suppose so we didn't rely on the `java.util.TimeZone.getDefault()`.

`./sbt` also explicitly sets the timezone to Sydney so helpful in `DEV` too...